pkg/dashboard/list.js

```diff
@@ -55,24 +57,26 @@ var common_plot_options = {
 
 var resource_monitors = [
     { selector: "#dashboard-plot-0",
-      plot: {
-          direct: [
-              "kernel.all.cpu.nice",
-              "kernel.all.cpu.user",
-              "kernel.all.cpu.sys"
-          ],
-          internal: [
-              "cpu.basic.nice",
-              "cpu.basic.user",
-              "cpu.basic.system"
-          ],
-          units: "millisec",
-          derive: "rate",
-          factor: 0.1  // millisec / sec -> percent
+      plot: function (machine_info) {
```

Comment:
When reading that first, it was rather confusing for `machine_info` now both being the globally imported function from machine-info.jsx *and* a local parameter here.

Reply:
Yes, me too.  Fixed.